posthog/hogql_queries/query_runner.py

```diff
@@ -1,14 +1,14 @@
 from abc import ABC, abstractmethod
```

Comment:
NB: changes in this file are all no-op and come from an auto-formatter I have installed in my editor.  Perhaps it's something I should turn off.